### DISCUSSION TITLE
Why notification is not working sometimes in self hosted server?

### DISCUSSION THREAD

**Discussion by perperj-super** [RocketChat/Rocket.Chat#32473] [Q&A]
Hello!

I self-hosted rocket chat and made everything working. However the painful thing is sometimes notification is not working well, so I can't know if someone sends me a message.
I need to restart the app or reload the server several times and it started working.

Any ideas on this?

Thanks

**Comment by alisharify7** [comment]
i have same issue i self host 
rocket chat and register it in cloud but its not sending notification to users in mobile application

<img width="1574" height="173" alt="rocket" src="https://github.com/user-attachments/assets/33123a06-7f03-43e1-a282-342c6c7a9c99" />

**Comment by reetp** [comment]
No one can help if you don't ask well documented questions.

[How to ask for help](https://gist.github.com/reetp/a66149d5f060f260643a353ca7067a98#before-you-start)

https://open.rocket.chat/channel/support
